migrations/2021-09-20-112945_jwt-secret/up.sql

```diff
@@ -0,0 +1,10 @@
+-- generate a jwt secret with 62 possible characters and length 43.
+-- this gives an entropy of 256 bits
+-- log2(62^43) = 256
+
+create table secrets(
+  id serial primary key,
+  jwt_secret varchar(43) not null
+);
+-- TODO: generate a random string from A-Za-z0-9, length 43, and insert
+insert into secrets(jwt_secret) values('123');
```

Comment:
@dessalines Could you tell me how to generate a random string here? I found [some examples](https://stackoverflow.com/questions/36533429/generate-random-string-in-postgresql), but dont know how to insert it instead of simply printing. My SQL is so rusty that I even had to look up the syntax for create and insert 😅

Reply:
This one has the best https://stackoverflow.com/a/41608000/1655478

Seems like you're not going to get better than a UUID for randomness in native postgres. You could always double it for something that's way beyond 128 bit.